### ISSUE TITLE
aptitude

### ISSUE THREAD

**Issue by casperdcl** [tqdm/tqdm#247]
Would be great if we can get this in aptitude land. See https://launchpad.net/ubuntu/+source/tqdm/4.7.4-1

**Comment by CrazyPython** [comment]
@casperdcl You have no aptitude to get it on aptitude.

**Comment by casperdcl** [comment]
On this topic, can we get this in the awesome list? We just need 20 votes, I think. Can everyone (incl. @tqdm/maintainers) :+1: [this](https://github.com/vinta/awesome-python/pull/662)?

**Comment by hadim** [comment]
Done

**Comment by casperdcl** [comment]
Hey @hadim, thanks. Nice to see you back! Also thanks @kmike. That was quick!

**Comment by CrazyPython** [comment]
@casperdcl yeah, tqdm really is awesome. I mean everybody would want progress bars on everything with the simplicity tqdm! Progress bar for internals! Progress bars for externals! Everybody gets a progress bar!

**Comment by CrazyPython** [comment]
@casperdcl watcha put in tqdm/maintainers? Illuminati mole? Watcha hiding?

**Comment by hadim** [comment]
And thank you guys for maintaining this great library tqdm. You're doing a nice job !

**Comment by casperdcl** [comment]
Hey, we were accidentally responsible for https://twitter.com/WikimediaItalia/status/914448810117545985
Also https://github.com/search?q=tqdm&type=Code
and http://www.metafilter.com/161265/An-alternative-to-an-eternally-spinning-wheel#6644017
and http://blog.rtwilson.com/my-top-5-new-python-modules-of-2015/
